proto/viam/app/packages/v1/packages.proto

```diff
@@ -110,6 +110,7 @@ message GetPackageRequest {
   string name = 2;
   string version = 3;
   string id = 4;
+  optional bool include_url = 5;
```

Comment:
Could you make sure app tests run fine while pointing to this API change? When I added `id` yesterday, it was required to be passed along in the corresponding UI files (.vue and .ts) since it was non-optional — I assume this _won't_ need to be added since it's explicitly optional.

I would suggest keeping this field as optional in this PR to maintain consistency within the current GetPackageRequest, and if we decide to move everything away from optional, make that breaking migration change altogether.

Relating to the discussion yesterday, we do use optional in other places to indicate that it's not required vs treating the null: e.g. https://github.com/viamrobotics/api/blob/a0c78972c8fcc21eb5ffff00e30b67cebe748bb1/proto/viam/app/v1/robot.proto#L37 and https://github.com/viamrobotics/api/blob/756c4acd5aa375b620ee26dcba13c5b6e03f7acd/proto/viam/app/v1/app.proto#L362 (though the code does check for nil here, but an empty name or location ID is invalid)

Reply:
Just a small suggestion to include a comment defining the default behavior for GetPackage and ListPackages.
Right now, it reads `// GetPackage returns the URL and metadata for a requested package version`,  and something similar for `ListPackages` which IMO seems like URL are returned by default. However, iiuc from what was said above, the default behavior, if `include_url` is NOT specified is to not return the URL.

Reply:
Thanks for the explanation Alexa! Btw I ran all the app tests locally and they passed.

Good catch Tahiya - I just updated those comments to make it clear that the URLs are only returned if requested!